internals/types/index.d.ts

```diff
@@ -353,7 +353,11 @@ export interface Project extends ProjectSettings {
   link?: ProjectLinkData;
   alias?: ProjectAliasTarget[];
   latestDeployments?: Partial<Deployment>[];
-  lastRollbackTarget: RollbackTarget | null;
+  lastAliasRequest?: LastAliasRequest;
+  /**
+   * @deprecated - Use `lastAliasRequest` instead
+   */
+  lastRollbackTarget?: RollbackTarget;
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏can `lastRollbackTarget` still be `null` sometimes?

Reply:
Yeah, I think `api` could return `null`. Better safe than sorry. Fixed!